### PR TITLE
(DE-908) docs(dbt): Add dbt tables doc for training_data

### PR DESCRIPTION
**Pull request by lmontier-pass**
# DE PR

## Describe your changes

Please include a summary of the changes:

* This PR [adds/removes/fixes/replaces] the [feature/bug/etc].
* Tag a reviewer if necessacy  @github/username

## Jira ticket number and/or notion link

JIRA-ticket_number

### Type of change

* [ ] Fix (non-breaking change which corrects expected behavior)
* [ ] New fields (non-breaking change)
* [ ] New table (non-breaking change)
* [ ] Concept change (potentially breaking change which modifies fields according to new or evolving business concepts)
* [ ] Table deletion (potentially breaking change which adds functionality/ table)

### Checklist before requesting a review

* [ ] I have performed a self-review of my code

* [ ] My code passes CI/CD tests
* [ ] I updated README.md
* [ ] I have updated the dag
* [ ] If my changes concern incremental table, I have altered their schema to accomodate with field's creation/deletion
* [ ] I have made corresponding changes to the [tables documentation](https://www.notion.so/passcultureapp/Documentation-Tables-175a397a8e854ff4a55ae4f3620dbe3b)
* [ ] I have made corresponding changes to the [fields glossary](https://www.notion.so/passcultureapp/854a436a8f1541e1b6ec2a65f8bab600?v=798024ba90404b139e5a17407a3bc604)
* [ ] I will create a review on slack and ensure to specify the duration of the review task: short (<10min), medium (<30min), long (>30min)

### Added tests?

* [ ] 👍 yes
* [ ] 🙅 no, because they aren't needed
* [ ] 🙋 no, because I need help
* [ ] ⏰ no, but I created a ticket

### PR title format (except for MEP)

There is a linter on the PR title format. Please respect the following format:

<details>
<summary>(ticket) type(topic): comment</summary>

* ticket surrounded by parenthesis, with optionnaly a hyphen followed by one or more digits (e.g., -1234). The first part must be one of the following strings:
  * DA
  * DE
  * AE
  * DS
  * HF
  * BSR
  * PC

* type :
The second part to specify the type of change one of the following :
  * build
  * lint
  * ci
  * docs
  * feat
  * fix
  * perf
  * refactor
  * test
  * core
  * dbt

* topic within parenthesis: 1 word e.g., (dag)

* comment: tell us your life

examples:

* :white_check_mark: (DE-124) refactor(firebase): update source field
* :x: (DE-124) refactor (firebase): update source field **(space between type and topic)**
* :x: (DE-124) airflow(firebase): update source fiedd in DAG **(wrong type)**
* :x: (DE-124) (DE-124) refactor(firebase refacto): update source field **(topic in two words)**
* :white_check_mark: (BSR) docs(github): add PR title valid format in template

</details>

### CODE REVIEW

File: orchestration/dags/data_gcp_dbt/models/_dbt_docs/glossary/applicative_database/column__metric.md [hunk 1]
```diff
@@ -20,60 +21,87 @@ title: Metrics
 {% docs column__total_created_individual_offers %} Total number of individual offers created. {% enddocs %}
 {% docs column__total_created_offers %} Total number of offers created, both individual and collective. {% enddocs %}
 {% docs column__total_credited_beneficiaries %}The total number of credited beneficiaries.{% enddocs %}
-{% docs column__total_current_year_collective_real_revenue %}The total real revenue from collective bookings for the current year.{% enddocs %}
-{% docs column__total_current_year_non_cancelled_tickets %} Total number of tickets that were not cancelled in the current year. {% enddocs %}
+{% docs column__total_current_year_collective_real_revenue %}The total real revenue from collective bookings for the
```
Comment (vbusson-pass) [inline orchestration/dags/data_gcp_dbt/models/_dbt_docs/glossary/applicative_database/column__metric.md:24]: real ? apres application du barème de remboursement ?

File: orchestration/dags/data_gcp_dbt/models/_dbt_docs/glossary/applicative_database/column__metric.md [hunk 2]
```diff
@@ -20,60 +21,87 @@ title: Metrics
...
+{% docs column__total_current_year_non_cancelled_tickets %} Total number of tickets that were not cancelled in the
+current year. {% enddocs %}
 {% docs column__total_current_year_real_revenue %} Total real revenue for the current year. {% enddocs %}
 {% docs column__total_current_year_tickets %}The total number of tickets for the current year.{% enddocs %}
-{% docs column__total_current_year_non_cancelled_collective_bookings %} The total non cancelled collective bookings for the current year. {% enddocs %}
-{% docs column__total_current_year_collective_theoretic_revenue %} Total collective theoretical revenue for the current year. {% enddocs %}
-{% docs column__total_current_year_used_collective_bookings %} Total collective used bookings for the current year. {% enddocs %}
+{% docs column__total_current_year_non_cancelled_collective_bookings %} The total non cancelled collective bookings for
+the current year. {% enddocs %}
+{% docs column__total_current_year_collective_theoretic_revenue %} Total collective theoretical revenue for the current
```
Comment (vbusson-pass) [inline orchestration/dags/data_gcp_dbt/models/_dbt_docs/glossary/applicative_database/column__metric.md:32]: theoretic ? -> expected revenue ?

File: orchestration/dags/data_gcp_dbt/models/_dbt_docs/glossary/applicative_database/column__metric.md [hunk 3]
```diff
@@ -20,60 +21,87 @@ title: Metrics
...
+year. {% enddocs %}
+{% docs column__total_current_year_used_collective_bookings %} Total collective used bookings for the current year. {%
```
Comment (vbusson-pass) [inline orchestration/dags/data_gcp_dbt/models/_dbt_docs/glossary/applicative_database/column__metric.md:34]: Total collective bookings used for the current year.

File: orchestration/dags/data_gcp_dbt/models/_dbt_docs/glossary/applicative_database/column__metric.md [hunk 4]
```diff
@@ -20,60 +21,87 @@ title: Metrics
...
-{% docs column__total_non_cancelled_collective_bookings %} Total number of collective bookings that were not cancelled. {% enddocs %}
-{% docs column__total_non_cancelled_duo_bookings %} Total number of non-cancelled duo bookings made by the user. {% enddocs %}
-{% docs column__total_non_cancelled_individual_bookings %} Total number of non-cancelled individual bookings made by the user. {% enddocs %}
+{% docs column__total_last_deposit_digital_goods_amount_spent %} Total amount spent on digital goods from the last
+deposit. {% enddocs %}
+{% docs column__total_non_cancelled_bookings %} Total number of bookings that were not cancelled, both individual and
+collective. {% enddocs %}
+{% docs column__total_non_cancelled_collective_bookings %} Total number of collective bookings that were not cancelled.
+{% enddocs %}
+{% docs column__total_non_cancelled_duo_bookings %} Total number of non-cancelled duo bookings made by the user. {%
```
Comment (vbusson-pass) [inline orchestration/dags/data_gcp_dbt/models/_dbt_docs/glossary/applicative_database/column__metric.md:57]: rajouter un commentaire (duo -> individual booking only)

File: orchestration/dags/data_gcp_dbt/models/_dbt_docs/glossary/applicative_database/column__metric.md [hunk 5]
```diff
@@ -20,60 +21,87 @@ title: Metrics
...
+enddocs %}
+{% docs column__total_non_cancelled_individual_bookings %} Total number of non-cancelled individual bookings made by the
```
Comment (vbusson-pass) [inline orchestration/dags/data_gcp_dbt/models/_dbt_docs/glossary/applicative_database/column__metric.md:59]: duo, included ?

File: orchestration/dags/data_gcp_dbt/models/_dbt_docs/glossary/applicative_database/column__metric.md [hunk 6]
```diff
@@ -20,60 +21,87 @@ title: Metrics
...
 {% docs column__total_stock_quantity %} Total remaining stock quantity. {% enddocs %}
 {% docs column__total_students %}The total number of students for the educational institution.{% enddocs %}
 {% docs column__total_theoretic_revenue %} Total theoretical revenue from all bookings. {% enddocs %}
-{% docs column__total_theoretical_amount_spent %} Total theoretical amount that would have been spent based on expected transactions. {% enddocs %}
-{% docs column__total_theoretical_amount_spent_in_digital_goods %} Total theoretical amount spent on digital goods based on expected transactions. {% enddocs %}
-{% docs column__total_theoretical_digital_goods_amount_spent %} Total theoretical amount spent on digital goods by the user. {% enddocs %}
-{% docs column__total_theoretical_outings_amount_spent %} Total theoretical amount spent on outings by the user. {% enddocs %}
-{% docs column__total_theoretical_physical_goods_amount_spent %} Total theoretical amount spent on physical goods by the user. {% enddocs %}
+{% docs column__total_theoretical_amount_spent %} Total theoretical amount that would have been spent based on expected
+transactions. {% enddocs %}
```
Comment (vbusson-pass) [inline orchestration/dags/data_gcp_dbt/models/_dbt_docs/glossary/applicative_database/column__metric.md:74]: un peu cryptique: expected transaction -> est ce que ca signifie all booking cancelled included ?)

File: orchestration/dags/data_gcp_dbt/models/_dbt_docs/glossary/tracking/column__event.md [hunk 7]
```diff
@@ -1,42 +1,58 @@
...
+enddocs %}
 {% docs column__user_web_browser %}The web browser used by the user, such as Chrome, Firefox, or Safari.{% enddocs %}
 {% docs column__user_web_browser_version %}The version of the web browser used by the user.{% enddocs %}
 {% docs column__page_location %}The location of the page where the event occurred, typically a URL or path.{% enddocs %}
 {% docs column__url_path_extract %}The extracted path from the URL where the event occurred.{% enddocs %}
 {% docs column__page_referrer %}The referrer page that led to the current page where the event occurred.{% enddocs %}
 {% docs column__page_number %}The number of the page in a sequence, if applicable.{% enddocs %}
+{% docs column__event_type %}The type of the interaction event.{% enddocs %}
+{% docs column__event_hour %}The hour of the interaction event.{% enddocs %}
+{% docs column__event_day %}The day of the interaction event.{% enddocs %}
```
Comment (vbusson-pass) [inline orchestration/dags/data_gcp_dbt/models/_dbt_docs/glossary/tracking/column__event.md:54]: type ?

File: orchestration/dags/data_gcp_dbt/models/_dbt_docs/glossary/tracking/column__event.md [hunk 8]
```diff
@@ -1,42 +1,58 @@
...
+{% docs column__event_month %}The month of the interaction event.{% enddocs %}
```
Comment (vbusson-pass) [inline orchestration/dags/data_gcp_dbt/models/_dbt_docs/glossary/tracking/column__event.md:55]: type?

**Review by lrainteau-pass — APPROVED** [review]

**Pull request merged**